VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        public static string ChangePassword = "PassWord917231";
```

Comment:
[CODE_SMELL] Change the visibility of 'ChangePassword' or make it 'const' or 'readonly'.